### ISSUE TITLE
Display credential issuer's name 

### ISSUE THREAD

**Issue by andrea-dintino** [ForkbombEu/wallet#216]
to show the issuer's name, you get a new object the script qr_to_well-known produces a third object called 
```json
"credential_issuer_information": {
	"credential_issuer": "https://issuer1.zenswarm.forkbomb.eu/credential_issuer",
	"credential_endpoint": "https://issuer1.zenswarm.forkbomb.eu/credential_issuer/credential",
	"authorization_servers": [
		"https://authz-server1.zenswarm.forkbomb.eu/authz_server"
	],
	"display": [
		{
			"name": "DIDroom_Issuer1",
			"locale": "en-US"
		}
	],
	"jwks": {
		"keys": [
			{
				"kid": "did:dyne:sandbox.genericissuer:GPgX3sS1nNp7fgLWvvTSw4jUaEDLuBTNq5eJhvkVD9ER#es256_public_key",
				"crv": "P-256",
				"alg": "ES256",
				"kty": "EC"
			}
		]
	}
}
```
You need to take :  "display":  >  "name": "DIDroom_Issuer1"

**Comment by andrea-dintino** [comment]
Support added in https://github.com/ForkbombEu/mobile_zencode/pull/14

**Comment by phoebus-84** [comment]
Thanks! closed in #247
